opnsense_api/interfaces/loopback_controller.py

```diff
@@ -14,7 +14,7 @@ class LoopbackInterface(OPNsenseItem):
 class LoopbackController(OPNsenseItemController[LoopbackInterface]):
 
     def __init__(self, device):
-        super().__init__(device, "interfaces", "loopback_settings")
+        super().__init__(device, "interface", "loopback_settings")
```

Comment:
FYI, it was `interfaces` because that's the name of the module in the [API](https://docs.opnsense.org/development/api/core/interfaces.html). The first argument is module, second is controller. Those two values are used when formatting the API request.

Reply:
Yeah, i have no idea why my IDE changed that. Definitely wasn't a conscious decision on my part 😂. I'll change it back once I get to these controllers.